package.json

```diff
@@ -33,11 +33,13 @@
     "circular-json": "^0.1.6",
     "crypto": "^0.0.3",
     "es6-promise": "^2.0.0",
+    "forge-min": "^0.6.20",
```

Comment:
I haven't come across forge before!

The cloud social provider in here has been using ssh2 to establish SSL tunnels -- and ssh2 can generate public/private key pairs, too (actually ssh2-streams, one of its dependencies). This is fine for now but we should consider using one or the other to reduce bloat (and, if we do, the social provider may want to use the `Provisioner` class directly to avoid another browserify step).